cmd_move.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"log"
+
+	cloudstore "cloud.google.com/go/storage"
+	"github.com/emicklei/tre"
+	cloudkms "google.golang.org/api/cloudkms/v1"
+)
+
+// command_move transfers a secret from a source to a target profile.
+func command_move(
+	kmsService *cloudkms.Service,
+	storageService *cloudstore.Client,
+	source profile,
+	sourceKey string,
+	target profile,
+	targetKey string) {
+
+	// fetch value for key from source
+	sourceValue, err := getValueByKey(kmsService, storageService, sourceKey, source)
+	if err != nil {
+		log.Fatal(tre.New(err, "get source key failed", "key", sourceKey))
+	}
+	// store value for key to target
+	command_put_paste(kmsService, storageService, target, "put", targetKey, sourceValue)
```

Comment:
So this could fail as well, right? For example, a situation where you have permissions on source, but not on target. Might be good to also log something here.

Reply:
the command_put_paste is taking care of this.